api.js

```diff
@@ -122,7 +124,13 @@ var v2 = {
             emitter = new events.EventEmitter();
         }
 
-        var req = http.request(options, function(res) {
+        let urlinfo = url.parse(config.standup.url);
+        options.host = urlinfo.hostname;
+        options.port = urlinfo.port;
+        options.protocol = urlinfo.protocol;
+        var request = (options.protocol === 'https:' ? https : http).request;
```

Comment:
Out of curiosity, why use let on line 127 and var here and in other places? Does it affect something I'm not seeing?